cobbler/modules/managers/isc.py

```diff
@@ -127,12 +119,11 @@ def write_v4_config(self):
                     host = system.interfaces[interface["interface_master"]]["dns_name"]
 
                     if ip is None or ip == "":
-                        # TODO: Clarify variable names
-                        for (nam2, int2) in list(system.interfaces.items()):
-                            if nam2.startswith(interface["interface_master"] + ".") \
-                                    and int2["ip_address"] is not None \
-                                    and int2["ip_address"] != "":
-                                ip = int2["ip_address"]
+                        for (interface_name, interface_object) in list(system.interfaces.items()):
+                            if interface_name.startswith(interface["interface_master"] + ".") \
+                                    and interface_object.ip_address is not None \
+                                    and interface_object.ip_address != "":
+                                ip = interface_object.ip_address
```

Comment:
I see you change the way of accessing the attributes for the interface object, now it seems these are object properties.

I just wonder if that Is the case in all cases? Was this broken before?

Reply:
I already answered myself on this after reading the full commit message. :smile: